draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -580,6 +580,78 @@ To ensure readability and replicability, the examples in this specification do
 not contain decoy digests unless explicitly stated. For an example
 with decoy digests, see (#example-simple_structured).
 
+### Recursive Disclosures {#recursive_disclosures}
+
+The algorithms above are compatible with "recursive disclosures", in which one
+selectively disclosed field reveals the existence of more selectively
+disclosable fields.  For example, consider the following JSON structure:
+
+```json
+{
+    "family_name": "Möbius",
+    "nationalities": ["DE", "FR", "UK"]
+}
+```
+
+When the Holder has multiple nationalities, the issuer may wish to allow the holder to reveal
+each of those nationalities individually.
+This can be accomplished by first making the "FR" and "UK" entries within
+the array selectively disclosable, and then the whole "nationalities" field.
```

Comment:
```suggestion
This can be accomplished by first making the entries within the "nationalities"
array selectively disclosable, and then the whole "nationalities" field.
```

Reply:
as kinda noted in 02d2fb07001057bbbe9532248478f5d31de80718 the text still needs to be adjusted and aligned to what the examples have but this is directionally correct.